XamarinCommunityToolkit/XamarinCommunityToolkit.csproj

```diff
@@ -72,9 +66,7 @@
     <Compile Include="**/*.shared.*.cs" />
     <None Include="..\LICENSE" PackagePath="" Pack="true" />
     <Folder Include="Behaviors\" />
-    <Folder Include="Controls\" />
```

Comment:
Why are these folders removed? I would expect Controls to still be around as `AvatarView` should be in there.

Reply:
There is another one "Views"